.github/workflows/cypress.yml

```diff
@@ -64,11 +64,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        browsers: [chrome, firefox, electron]
         node-version: [16.x]
         include:
           - browsers: electron
             spec: cypress/e2e/default/learn/challenges/projects.js
+          - browsers: chrome
+            spec: cypress/e2e/default/**/*.js
+          - browsers: firefox
+            spec: cypress/e2e/default/**/*.js
```

Comment:
I might be wrong, but @Nirajn2311 had said something about the spec not getting picked up from actions. Correct?

Reply:
He can confirm, but my understanding was that the list of tests gets filtered by the config and then again by the `--spec` parameter, if it exists.  We could try not passing it for chrome and firefox, but I wasn't sure how to do that.  @naomi-lgbt any ideas?

Reply:
```yml
if ${{ matrix.browsers}} === 'electron'
```

Maybe?

Reply:
Do you want push a commit if it looks cleaner?

Reply:
TM, indeed.